.env.example

```diff
@@ -0,0 +1,15 @@
+# network specific node uri : `"ETH_NODE_URI_" + networkName.toUpperCase()`
```

Comment:
The intention of this file is to have an easy way to keep ENV variables accessible (e.g. mnemonic, nodes, secrets)